sentry_telegram/plugin.py

```diff
@@ -127,7 +127,7 @@ def send_message(self, url, payload, receiver):
         )
         self.logger.debug('Response code: %s, content: %s' % (response.status_code, response.content))
 
-    def notify_users(self, group, event, fail_silently=False):
+    def notify_users(self, group, event, triggering_rules, fail_silently=False):
```

Comment:
Hello, @pupaxxo!
Thank you for commit and for search the reason for incompatibility 👍

Actually, this change brakes plugin compatibility with Sentry < 9.1. You can see that in tests result.
Also, the new argument doesn't use in my plugin.
Therefore, in that case, I prefer to add `**kwargs` into signature instead of position argument.

Sentry promised a lot of backward incompatibilities at following releases, so, maybe in future versions, we will change the approach to work with `triggering_rules` :)

```suggestion
    def notify_users(self, group, event, fail_silently=False, **kwargs):
```

Reply:
👍  That's a better solution, I can't "python" so I just typed the first (and fastest) solution to fix the plugin (we mainly use telegram for notifications, we can't live without it 😅)